src/scss/style.scss

```diff
@@ -0,0 +1,92 @@
+.module {
+    color: #625F5F;
+    border-radius: 2px;
+    padding: 15px;
+}
+
+body {
```

Comment:
It is a good practice to place the common css styles on top. 
Also, a well-proven practice is to use styling like normalize or reset css.